gcloud/_helpers.py

```diff
@@ -439,8 +441,7 @@ def _to_bytes(value, encoding='ascii'):
     :rtype: str / bytes
     :returns: The original value converted to bytes (if unicode) or as passed
               in if it started out as bytes.
-    :raises: :class:`TypeError <exceptions.TypeError>` if the value
-             could not be converted to bytes.
+    :raises TypeError: if the value could not be converted to bytes.
```

Comment:
Does it allow for the `exc` role in the text of `:raises:`?  e.g.

```
`:raises: :exc:`TypeError`
```

I find that syntax preferable to `:class:` for exceptions.